src/screens/home/ducks/home.actions.js

```diff
@@ -0,0 +1,10 @@
+
+import * as HomeActionTypes from "./home.types";
+
+ export function addContact(user, data) {
+    return {
+      type: HomeActionTypes.ADD_CONTACT_SUCCESS,
+      user,
```

Comment:
here, we can use a single key named "payload" and inside the payload, we can use different data keys. It will help us in maintaining the uniformity of all the action creators. As "payload" is a redux term which means some data is inside that key.